### PR TITLE
initial comment; rbac-lookup

### PR DESCRIPTION
**Pull request by dualfade**
Please review. Thank you !

### CODE REVIEW

File: packages/rbac-lookup/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+license=('Apache')
+depends=()
+makedepends=('git' 'go')
+source=("git+https://github.com/FairwindsOps/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo "$(git rev-list --count HEAD).$(git rev-parse --short HEAD)"
```
Comment (noraj) [inline packages/rbac-lookup/PKGBUILD:20]: git tags available, please use the tags version

File: packages/rbac-lookup/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    -o $pkgname .
+}
+
+package() {
+  cd $pkgname
+
+  install -Dm 755 $pkgname "$pkgdir/usr/bin/$pkgname"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (noraj) [inline packages/rbac-lookup/PKGBUILD:42]: Don't copy for generic license https://wiki.archlinux.org/title/PKGBUILD#license

It's actually mentioned in the template https://github.com/BlackArch/blackarch-pkgbuilds/blob/ed1aab7282e0d4143d3dfc62304376a3c946e705/PKGBUILD-go#L49-L51

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/rbac-lookup/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/rbac-lookup/PKGBUILD:20]: Will fix.

File: packages/rbac-lookup/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/rbac-lookup/PKGBUILD:42]: My bad; will fix.

**Pull request merged**